src/rpc/blockchain.cpp

```diff
@@ -1583,14 +1583,14 @@ UniValue savemempool(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() != 0) {
         throw std::runtime_error(
             "savemempool\n"
-            "\nDumps the mempool to disk.\n"
+            "\nDumps the mempool to disk. It will fail until the previous dump is fully loaded.\n"
```

Comment:
this also cannot complete when `-persistmempool` is false, yes? Might want to add that to RPC notes, or set it to true always if persist is false.

Reply:
I think it's probably important that this works if `-persistmempool` is false.

Reply:
Indeed, definitely needs to work with -persistmempool false, that's the whole point of the savemempool command. AFAICT the current implementation does work in that case.

Reply:
I think it's best it works in that case, just didn't thought about it initially. But for the record, I'm using savemempool with -persistmempool to true, because I want to read it on restart too!
I think the whole point of savemempool is to cover use cases wher for whatever reason you can't or don't always let the daemon exit gracefully.